.gitignore

```diff
@@ -29,7 +29,6 @@ CATcher.iml
 
 # IDE - VSCode
 .vscode/*
-!.vscode/launch.json
```

Comment:
Is this related to #888 and #876? If I'm not wrong since #876 was closed we're not including `launch.json` in `gitignore`?

Reply:
Accidentally added this change, thanks for pointing out